packages/mdc-select/foundation.js

```diff
@@ -153,9 +152,9 @@ export default class MDCSelectFoundation extends MDCFoundation {
     if (this.selectedIndex_ >= 0) {
       selectedTextContent = this.adapter_.getTextForOptionAtIndex(this.selectedIndex_).trim();
       this.adapter_.setAttrForOptionAtIndex(this.selectedIndex_, 'aria-selected', 'true');
-      this.adapter_.addClassToLabel(cssClasses.LABEL_FLOAT_ABOVE);
+      this.adapter_.floatLabel(true);
```

Comment:
We looked up the spec yesterday and saw that this should be moved to the open function, and line 157 should be moved to the closed function.